packages/backend-defaults/src/entrypoints/rootHttpRouter/http/MiddlewareFactory.test.ts

```diff
@@ -254,5 +244,57 @@ describe('MiddlewareFactory', () => {
 
       expect(childLogger.error).toHaveBeenCalled();
     });
+
+    it('should log incoming requests', async () => {
+      const app = express();
+      app.use(middleware.logging());
+      app.get('/', (_req, res) => res.send('Hello World'));
+
+      await request(app).get('/').expect(200);
+
+      expect(logger.info).toHaveBeenCalledWith(
+        expect.stringContaining(
+          '[20/Nov/2024:00:00:00 +0000] "GET / HTTP/1.1" 200 11 "-" "-"',
+        ),
+        {
+          type: 'incomingRequest',
+          date: '20/Nov/2024:00:00:00 +0000',
+          method: 'GET',
+          url: '/',
+          status: '200',
```

Comment:
could this be a number?

Reply:
I converted to number, people probably expect that 🙂